src/Application/Actions/ChangePasswordUsingToken.php

```diff
@@ -86,6 +88,7 @@ protected function action(Request $request, array $args): Response
         }
 
         $token->consume(new \DateTimeImmutable());
+        $person->email_address_verified = $this->now;
```

Comment:
not strictly related to other stuff in the PR but will be good to know that users who've reset a password this way have also verified their email addresses.